x11/windowmaker/Portfile

```diff
@@ -38,26 +48,20 @@ depends_lib \
 	port:giflib \
 	port:tiff
 
-post-patch {
-	set fl [glob -directory ${worksrcpath} WindowMaker/*menu*]
-	lappend fl ${worksrcpath}/WindowMaker/Defaults/WindowMaker.in \
-	               ${worksrcpath}/WPrefs.app/Menu.c ${worksrcpath}/WPrefs.app/Paths.c
-	foreach i $fl {
-		reinplace "s|/usr/X11R6/share|${prefix}/share|g" $i
-		reinplace "s|/usr/share|${prefix}/share|g" $i
-		reinplace "s|/usr/local/share|${prefix}/share|g" $i
-		reinplace "s|/opt/share|${prefix}/share|g" $i
-		reinplace "s|/opt/kde2/share|${prefix}/share|g" $i
-	}
-}
```

Comment:
I don't think it's correct to remove all of these reinplaces. Some of these files, such as WPrefs.app/Menu.c, still contain references to some of those paths and need to have them changed; I did not check all the files.